src/main/riddl/Commerce/commerce.riddl

```diff
@@ -8,23 +8,6 @@ context Commerce is {
         action: String
     }
     type PaymentId is Id( Payment )
-    type LineItem is {
-        productId: ProductId,
-        quantity: Number,
-        promotion: PromotionId,
-        extendedPrice: Number //Price of product after promotion has been applied
-        //TODO: At doTERRA we calculated tax at the LineItem Level
-    }
-    type OrderId is Id( Order )
-    type OrderInfo is {
-        customerId: MemberId, //TODO: This should be CustomerId but not sure how to model that in RIDDL
-        lineItems: LineItem+,
-        itemTotal: Number,
-        taxTotal: Number,
-        deliveryTotal: Number, //TODO: Should delivery just be a LineItem?
-        grandTotal: Number,
-        paymentId: PaymentId
-    }
     type DeliveryId is Id( Delivery )
```

Comment:
Why are we removing these? It doesn't look like you moved them anywhere, as no other file includes this information.  How is an Order supposed to be represented in this application?

Reply:
These are already modelled in Order form Kalix Study. I talked it over with James and he agreed to take them out of Commerce because Order is big enough for its own entity now

Reply:
Okay good enough for me.